src/tests/ftest/cart/test_group_np_cli.c

```diff
@@ -46,30 +91,56 @@ test_run(void)
 	crt_rpc_t		*rpc_req = NULL;
 	int			 i;
 	int			 rc = 0;
+	uint32_t		*_cg_ranks;
+	int			 _cg_num_ranks;
 
-	if (test_g.t_save_cfg) {
-		rc = crt_group_config_path_set(test_g.t_cfg_path);
-		D_ASSERTF(rc == 0, "crt_group_config_path_set failed %d\n", rc);
-	}
+	if (test_g.t_skip_init) {
+		DBG_PRINT("Skipping init stage.\n");
 
-	tc_cli_start_basic(test_g.t_local_group_name,
-			   test_g.t_remote_group_name,
-			   &grp, &rank_list, &test_g.t_crt_ctx[0],
-			   &test_g.t_tid[0], test_g.t_srv_ctx_num,
-			   test_g.t_use_cfg, NULL);
+	} else if (!test_g.t_skip_init) {
 
-	rc = sem_init(&test_g.t_token_to_proceed, 0, 0);
-	D_ASSERTF(rc == 0, "sem_init() failed.\n");
+		if (test_g.t_save_cfg) {
+			rc = crt_group_config_path_set(test_g.t_cfg_path);
```

Comment:
```suggestion
		if (test_g.t_save_cfg) {
			rc = crt_group_config_path_set(test_g.t_cfg_path);
			D_ASSERTF(rc == 0,
```

Reply:
This suggestion is odd in that it doesn't note what [checkpatch.pl](https://github.com/torvalds/linux/blob/master/scripts/checkpatch.pl) issue it's resolving.